release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -858,6 +858,14 @@ clouds:
 
 To use application credentials with your cluster as a {rh-openstack} administrator, create the credentials. Then, use them in a `clouds.yaml` file when you install a cluster. Alternatively, you can create the `clouds.yaml` file and rotate it into an existing cluster.
 
+[id="ocp-4-12-hcp"]
+=== Hosted control planes (Technology Preview)
+
+[id="ocp-4-12-hcp-versioning"]
+==== Versioning support for hosted control planes
+
```

Comment:
Thanks! I think this should read similar to:

With each major, minor, or patch version release of hypershift, 2 components of hosted control planes are released: the HyperShift Operator and the command-line interface (CLI). Any HyperShift Operator that is released for a minor _n_ version of {product-title} must support _n_, _n-1_, and _n-2_ versions of the Openshift release payload.
The HostedCluster and NodePool APIs are currently beta and follow similar policy to [Openshift](https://docs.openshift.com/container-platform/4.11/rest_api/understanding-api-support-tiers.html) and [Kubernetes](https://kubernetes.io/docs/concepts/overview/kubernetes-api/).

Reply:
Thanks @enxebre! Your revision sounds good to me, and it made me realize that I need to update the information in the product docs (which is set to publish when 4.12 is GA). Those docs were reviewed and approved, but they say that 3 components are released with each release of OCP.[1] However, when I just cross-referenced the upstream docs, it looks like you are correct. 

I'll make a to-do for myself to update that part of the docs. 

[1]: Product docs preview: https://redhat:redhat@docs.openshift.com/container-platform/4.12/architecture/control-plane.html#hosted-control-planes-version-support_control-plane